assets/witnet/addresses.js

```diff
@@ -1,5 +1,7 @@
-
+const { merge } = require("lodash")
```

Comment:
All lodash functions have their own npm package. Could we use [`lodash.merge`](https://www.npmjs.com/package/lodash.merge) instead of installing the whole `lodash` library (`lodash` 1.41 MB vs `lodash.merge` 54.1 kB)

Reply:
took note and released witnet-utils v1.0.9 as to import lodash.merge instead of lodash